src/main/java/com/thebois/models/beings/IBeing.java

```diff
@@ -45,4 +45,21 @@
      */
     Iterable<Position> getPath();
 
+    /**
+     * Returns true if the being is alive and false if it's dead.
+     *
+     * @return Whether the being is alive or not.
+     */
+    boolean isAlive();
+
+    /**
+     * Gets the health ratio, where 0 is dead and 1 is full health.
+     * <p>
+     * The Health ratio is current health divided by max health.
+     * </p>
```

Comment:
style

Right now it is, but that's an implementation detail which probably shouldn't be in the javadoc, right?